src/style.css

```diff
@@ -0,0 +1,89 @@
+body {
+  margin: 0;
+  height: 100%;
+}
+
+.container {
+  display: flex;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+}
+
+.container__block {
+  font-family: Arial;
+  color: white;
+  font-size: 100px;
+  flex-grow: 1;
+  height: 300px;
+  text-align: center;
+  min-width: 300px;
+  width: 33%;
+  align-self: center;
+  line-height: 300px;
+}
+
+.container__block_1 {
+  background-color: rgb(255, 0, 0);
+  order: 1;
+}
+
+.container__block_2 {
+  background-color: rgb(204, 0, 0);
+  order: 2;
+}
+
+.container__block_3 {
+  background-color: rgb(153, 0, 0);
+  order: 3;
+}
```

Comment:
you could not provide order value to elements that will not be change their default position. By default it will be 0, and they appear as in html order.

Reply:
Deleted unnecessary order properties in style.css